tables/vi-saigon-g1.ctb

```diff
@@ -1,5 +1,5 @@
 #-index-name: Vietnamese, Saigon, contracted
-#-display-name: Southern Vietnamese braille
+#-display-name: Vietnamese Saigon Braille
```

Comment:
I'm going to put this change in a separate commit. And change the "Braille" to lowercase.